.gitignore

```diff
@@ -32,6 +32,7 @@ vendor
 prebid-server
 build
 debug
+__debug_bin
```

Comment:
What tooling is writing to this path?

Reply:
That is the debug binary that Visual Studio Code is creating. I'm using 1.52.1. It is only present when debugging is actually running in the IDE.

Reply:
That's in the Linux version of Visual Studio Code.

Reply:
Ah ok. Thnx. I'm fine with it.